doc/source/user_guide/io.rst

```diff
@@ -28,6 +28,7 @@ The pandas I/O API is a set of top level ``reader`` functions accessed like
     :delim: ;
 
     text;`CSV <https://en.wikipedia.org/wiki/Comma-separated_values>`__;:ref:`read_csv<io.read_csv_table>`;:ref:`to_csv<io.store_in_csv>`
+    text;`TXT <https://www.oracle.com/webfolder/technetwork/data-quality/edqhelp/Content/introduction/getting_started/configuring_fixed_width_text_file_formats.htm>`__;:ref:`read_fwf<io.fwf_reader>`
```

Comment:
i might say fixed with text but not sure will fit

Reply:
@TomAugspurger Can you resolve?

Reply:
I'm not sure what you mean.

Reply:
@TomAugspurger He´s talking about "Format Type" on "IO Tools" table... I think that "Text" is Correct in read_fwf() case

Reply:
OK. Sounds fine.